userspace/libscap/engine/gvisor/CMakeLists.txt

```diff
@@ -21,6 +23,7 @@ find_package(Threads)
 target_link_libraries(scap_engine_gvisor 
     ${CMAKE_THREAD_LIBS_INIT}
     ${PROTOBUF_LIB}
+    ${JSONCPP_LIB}
```

Comment:
This feels so weird, i mean compiling the `JSONCPP_LIB_SRC` or linking the `JSONCPP_LIB`.
We should really address it in a subsequent PR :)

Reply:
This is because of the jsoncpp module we currently have! The next PR will address it

Reply:
Yep i well know it sadly :D